pom.xml

```diff
@@ -86,6 +86,11 @@
         <version>19.0</version>
       </dependency>
       <dependency>
+        <groupId>com.github.rholder</groupId>
+        <artifactId>guava-retrying</artifactId>
```

Comment:
Good to know this lib. We can replace own simple retry code (storage related) with this.